cluster/gce/gci/configure-helper.sh

```diff
@@ -55,6 +55,27 @@ function config-ip-firewall {
   if [[ "${ENABLE_METADATA_CONCEALMENT:-}" == "true" ]]; then
     iptables -A KUBE-METADATA-SERVER -j DROP
   fi
+
+  # Flush iptables nat table
+  iptables -t nat -F || true
+
+  if [[ "${NETWORK_POLICY_PROVIDER:-}" == "calico" && "${KUBERNETES_MASTER:-}" == false ]]; then
+    echo "Add rules for ip masquerade"
+    iptables -t nat -N IP-MASQ-AGENT
+    iptables -t nat -A POSTROUTING -m comment --comment "ip-masq-agent: ensure nat POSTROUTING directs all non-LOCAL destination traffic to our custom IP-MASQ-AGENT chain" -m addrtype ! --dst-type LOCAL -j IP-MASQ-AGENT
+    iptables -t nat -A IP-MASQ-AGENT -d 169.254.0.0/16 -m comment --comment "ip-masq-agent: local traffic is not subject to MASQUERADE" -j RETURN
+    iptables -t nat -A IP-MASQ-AGENT -d 10.0.0.0/8 -m comment --comment "ip-masq-agent: local traffic is not subject to MASQUERADE" -j RETURN
+    iptables -t nat -A IP-MASQ-AGENT -d 172.16.0.0/12 -m comment --comment "ip-masq-agent: local traffic is not subject to MASQUERADE" -j RETURN
+    iptables -t nat -A IP-MASQ-AGENT -d 192.168.0.0/16 -m comment --comment "ip-masq-agent: local traffic is not subject to MASQUERADE" -j RETURN
+    iptables -t nat -A IP-MASQ-AGENT -m comment --comment "ip-masq-agent: outbound traffic is subject to MASQUERADE (must be last in chain)" -j MASQUERADE
+  fi
+}
+
+function config-sysctls {
```

Comment:
Ike's PR will take care of route_localnet so you shouldn't need that, and kubelet will set bridge-nf-call-iptables so you should need that either, I would remove this function altogether.

Reply:
Done